test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/RazorDirectivesTest/BuiltInDirectiveDoesNotErorrIfNotAtStartOfLineBecauseOfWhitespace.cspans.txt

```diff
@@ -1,7 +1,7 @@
 Markup span at (0:0,0 [2] ) (Accepts:Any) - Parent: Markup block at (0:0,0 [26] )
-Code span at (2:1,0 [2] ) (Accepts:Any) - Parent: Directive block at (2:1,0 [24] )
```

Comment:
Can I fight for this? I don't get why the whitespace prior to a directive needs to be part of the directive. The way I designed it, the directive starts with a transition`@`.

Reply:
I'll let @NTaylorMullen weigh in on this.

Reply:
How do you plan to implement trivia for directives? In my mind the leading whitespace of a directive should have the following characteristics:
- Cannot contain any content
- Should have 0 intellisense/completion at design time
- Can be 0+ whitespace characters long.

With whatever trivia approach you have in mind would it make the above characteristics easy to implement/understand?

Reply:
> But don't we already allow comments before the start of a directive?

Nah 

![image](https://user-images.githubusercontent.com/2008729/46226598-a7179100-c311-11e8-8831-5f5fb1690f8e.png)

Do you think it makes sense to design (not implement) the plans for trivia now so we can make better choices with the SyntaxTree? Or does it make sense to wait and do reactive changes later?

Reply:
Spoke to @NTaylorMullen offline. We are not going to make this consistent with the old world because the old world already has different behavior depending on the contents of the previous line. I plan to keep this change and test this in the editor to make sure we didn't break anything.